src/notation/view/widgets/editstyle.ui

```diff
@@ -38,7 +38,7 @@
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
-      <enum>Qt::Horizontal</enum>
+      <enum>Qt::Orientation::Horizontal</enum>
```

Comment:
These changes (i.e. all those `Something::` additions) are introduced by Qt Creator 14, either intentionally or because of a bug. But they cause problems, mostly with vertical spacing.
This PR:
<img width="1057" alt="Scherm­afbeelding 2024-08-18 om 15 45 08" src="https://github.com/user-attachments/assets/0324b520-3831-44ca-8950-c687705e6198">
Master:
<img width="1045" alt="Scherm­afbeelding 2024-08-18 om 15 46 03" src="https://github.com/user-attachments/assets/0190cce9-d482-4f8b-a60e-01d57b4890d6">

(Ignore the different languages :) )

If you're familiar with how Qt compiles .ui files, you could try to find out where it goes wrong, i.e. why these changes cause the spacing to break. Otherwise, I'm afraid that the best thing you can do for now is to undo these changes manually. You can't do this in Qt Creator because it won't let you edit the file as a text file (only graphically), but you can open the file in any text editor like VS Code, and undo the changes there. Tip: leverage find&replace functionality. And beware, after you edit&save the file using Qt Creator 14 again, you'll have to undo all these changes again.